vmd-api/api/openapi.yaml

```diff
@@ -0,0 +1,336 @@
+openapi: 3.1.0
+info:
+  contact:
+    email: nikita.bazulin@unikie.com
+  description: This is Control Panel API based on the OpenAPI 3.1 specification of
+    the GHAF project
+  license:
+    name: Apache License 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  title: Virtual Machine Manager API of the GHAF project - OpenAPI 3.1
+  version: 1.0.1
+externalDocs:
+  description: Find out more about the GHAF project
+  url: https://github.com/tiiuae/ghaf
+servers:
+- description: The primary server
+  url: https://localhost/api/v1
+- description: The test server
+  url: https://localhost/api/test/v1
+tags:
+- description: Operations of Virtual Machines
+  name: vm
+paths:
+  /vm/list:
+    get:
+      description: Return a IDs list of all Virtual Machines
+      operationId: getVmList
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                items: {}
+          description: IDs list of all Virtual Machines
+        default:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+          description: error payload
+      summary: Get VMs list
+      tags:
+      - vm
+  /vm/info/{id}:
+    get:
+      description: Returns information about the Virtual Machine by ID
+      operationId: getVmInfoById
+      parameters:
+      - description: An ID of the Virtual Machine that needs to be fetched
+        explode: false
+        in: path
+        name: id
+        required: true
+        schema: {}
+        style: simple
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/VirtualMachineInfo'
+          description: successful operation
+        "400":
+          description: Invalid Virtual Machine ID supplied
+        "404":
```

Comment:
What is actually the semantic difference between 400 (invalid) and 404 (not found)?

Reply:
I had to guess, as well :-) 
I will be very explicit here. Let's think this OpenAPI specification as such that we give the reader (also future readers) these answers and definitions. Based on your comment, I'm guessing you agree with my guesses :-) So, would you please add the definitions of 404 (Invalid) and 404 (Not found) to the descriptions? Thank you.